ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java

```diff
@@ -201,126 +175,17 @@ public void onResult(@Nullable Configuration configuration, @Nullable Exception
         });
     }
 
-    /**
-     * Creates a stringified JSON object containing the information necessary to perform a lookup
-     *
-     * @param context Android Context
-     * @param request  the {@link ThreeDSecureRequest} that has a nonce and an optional UI customization.
-     * @param callback {@link ThreeDSecurePrepareLookupCallback}
-     */
-    public void prepareLookup(final Context context, final ThreeDSecureRequest request, final ThreeDSecurePrepareLookupCallback callback) {
```

Comment:
Is this method being replaced with something else? I believe it's used by merchants who do server-side lookups for 3DS, so I don't think it can go away entirely.

Reply:
Right yeah there doesn't seem to be an iOS equivalent for this method. We figured if it's requested we can always add back and potentially add it to iOS?

Reply:
Good catch - we added back the `prepareLookup` and `initializeChallengeWithLookupResponse` methods and did some refactoring for code reuse.